aep/general/0004/aep.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 id: 4
-slug: resource-types
```

Comment:
This one in particular seems a bit too generic - it overlaps (conceptually) with e.g. common component types.  (The others all seem fine.)

Reply:
I'll revert this one for now so we can get the others in - I'll send a follow-up to discuss this specifically.

My thought is for this AEP to be generic to a way to describe types - and common types could live under `common-types`.